stable/kommander-thanos/templates/secret.yaml

```diff
@@ -0,0 +1,33 @@
+{{ $thanosCa := genCA "Kommander Thanos CA" 3650 }}
+{{ $thanosServer := genSignedCert "server.thanos.localhost.localdomain" nil nil 3650 $thanosCa }}
+{{ $thanosClient := genSignedCert "client.thanos.localhost.localdomain" nil nil 3650 $thanosCa }}
+---
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: kommander-thanos-client-tls
```

Comment:
I'm using a fixed name for this secret because I need to pass it to the thanos subchart (`thanos.query.certSecretName`), and I wasn't sure how to use Helm-generated values in `values.yaml`.